relay/sources/relayd/Cargo.toml

```diff
@@ -19,45 +19,46 @@ harness = false
 
 [dependencies]
 #### Good
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-nom = "5.0"
+serde = { version = "1.0.98", features = ["derive"] }
```

Comment:
Why adding precise versions here? Semver should be enough, and this makes it a bit harder to upgrade direct dependencies, while providing no more control than Cargo.lock.